.gitattributes

```diff
@@ -8,6 +8,7 @@
 /vendor export-ignore
 
 # Files
+/.distignore
```

Comment:
Isn't the `export-ignore` missing here? The .distignore LGTM.

Reply:
Yes, you're right. Fixed with rebase.